src/Tmds.Ssh/Tmds.Ssh.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+   <FrameworkReference Include="Microsoft.AspNetCore.App" />
```

Comment:
fyi, this is to be able to use `System.IO.Pipelines` because it is not part of the .NET Core shared framework, but of the ASP.NET Core shared framework.